modules/microshift-cli-oc-about.adoc

```diff
@@ -11,3 +11,8 @@ With the OpenShift command-line interface (CLI), the `oc` command, you can deplo
 * Working directly with project source code
 * Scripting {product-title} operations
 * Managing projects while restricted by bandwidth resources
+
+[NOTE]
+====
+A `kubeconfig` file must exist for the cluster to be accessible. The values are applied from built-in default values or a `config.yaml`, if one was created.
```

Comment:
I have a misunderstanding of kubeconfig. Can I modify the contents of kubeconfig by making changes in config.yaml ?

Reply:
@dhensel-rh The application creates them at startup from the yaml, yes. https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/configuring/microshift-kubeconfig

Reply:
ok